front/app/containers/Admin/projects/project/events/index.tsx

```diff
@@ -125,14 +146,26 @@ const AdminProjectEventsIndex = ({
                   return (
                     <Row key={event.id}>
                       <div className="expand">
-                        <h1>
-                          <T value={event.attributes.title_multiloc} />
-                        </h1>
+                        <h1>{localize(event.attributes.title_multiloc)}</h1>
                         <p>{event.attributes.address_1}</p>
                         <p>
                           {startAt} → {endAt}
                         </p>
                       </div>
+                      <Box>
+                        <Title
+                          variant="h6"
+                          color="primary"
+                          fontSize="xl"
+                          p="0px"
+                          m="0px"
+                        >
+                          {event.attributes.attendees_count}
+                        </Title>
+                        <Text m="0px">
+                          <FormattedMessage {...messages.attendees} />
```

Comment:
I think if there is only 1 attendee it will say "1 attendees", but also that's super minor so I think that's probably fine :)